verification/src/http_server/routers/app.rs

```diff
@@ -0,0 +1,45 @@
+use super::{SolidityRouter, SourcifyRouter};
+use crate::{config::Config, http_server::handlers::status};
+use actix_web::web;
+
+pub struct AppRouter {
+    verification: Option<SolidityRouter>,
+    sourcify: Option<SourcifyRouter>,
+}
+
+impl AppRouter {
+    pub async fn new(config: Config) -> anyhow::Result<Self> {
+        let verification = match config.verifier.disabled {
```

Comment:
https://doc.rust-lang.org/std/primitive.bool.html#method.then

Reply:
It's nice idea, but seems that it may be a problem to `await` inside an `Option`. Not sure that it is applicable in our case, though try to research

Reply:
yeah, await probably won't work

Reply:
Applied that for the sourcify initialization (without `.await` call)